srcpkgs/apcupsd/files/apcupsd/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+install -d -m 0775 -o root -g uucp /run/apcupsd
```

Comment:
I'm unsure of this, too

Reply:
how would you check if a directory exists?

Reply:
also group `uucp`

Reply:
changed